examples/gce.js

```diff
@@ -39,19 +39,19 @@ th.run({
         -orchAddr https://orchestrator_0:8935 -orchSecret test'
     },
     orchestrators: {
-      instances: 14,
+      instances: 15,
       // TODO these are not complete, try adding the right orchestrator flags :)
-      flags: `-v 5 -initializeRound=true -gasPrice 200 -gasLimit 2000000 \
-      -currentManifest=true`
+      flags: `-v 5 -initializeRound=true -gasPrice 20 -gasLimit 20000000 \
+      -currentManifest=true -maxSessions 8`
     },
     broadcasters: {
       // uncomment to configure usage of Google Storage
       // googleStorage: {
       //   bucket: 'lptest-fran',
       //   key: 'examples/test-harness-226018-e3a05729b733.json'
       // },
-      instances: 16,
-      flags: `-v 5 -gasPrice 200 -gasLimit 2000000 \
+      instances: 15,
+      flags: `-v 5 -gasPrice 20 -gasLimit 20000000 \
```

Comment:
Just curious, do we have a sense of the # of streams that a broadcaster can comfortably handle given a standard GCE instance? Right now we seem to be running the same number of B/O, while I'd expect a B to handle ~4-5x the streams of a transcoding O. We don't have to change anything for this test setup; just wondering if these numbers are reflective of a bottleneck on B, or us playing it safe.

Reply:
well, it depends whether [this issue](https://github.com/livepeer/test-harness/issues/27) is the test-harness's fault or not. 
but if it's not LPMS's issue, I think the broadcaster can comfortably handle that level of streams compared to the transcoder. (4x or 5x)